VideoWeb/VideoWeb/ClientApp/src/app/services/models/new-allocation-message.ts

```diff
@@ -0,0 +1,6 @@
+export class NewAllocationMessage {
+    constructor(hearingDetails: []) {
```

Comment:
Suggest either typing this array or using a generic.

Reply:
ok changed

Reply:
you should be able to do something like 
``` class NewAllocationMessage<T> ... ```
and then use that to type the hearing details.
But it may be better to just not use a generic and have a HearingDetailsAllocationMessage or something.

Reply:
This might work:
```
class NewAllocationMessage<T>
```

and then
```
hearingDetails: Array<T>
```

Then, when using the model, you'll just have to specify a type for T, like this
```
const message = new NewAllocationMessage(hearingDetails);
```

Reply:
changed to strictly type object